herd/libdir/arm-models/mixed/ec.cat

```diff
@@ -40,17 +40,24 @@
 include "armv8-common.cat"
 include "sca.cat"
 
-let preorder-cb = IM0 | lob | scaob
+let preorder-cb = IM0 | local-hw-reqs | scaob
```

Comment:
I am curious is there a reason why we include all of `local-hw-reqs`? Isn't `lob` the subset of `local-hw-reqs` that applies to `Exp` events only?

Reply:
As you suggested elsewhere, some clauses in `local-hw-reqs` apply to `Exp` too. (Notably, `CSE-ob`.) That is the technical reason for using `local-hw-reqs`.

Conceptually, we need to use all externally observable orderings that contribute to `ob`, but do not depend on `rf` and `co`. In that sense, I'd prefer `local-hw-reqs` over `lob | CSE-ob`.